pendulum_control/test/execute_with_delay.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import subprocess
+import sys
+import time
+
+
+def main():
+    if len(sys.argv) < 3:
+        print("usage: execute_with_delay.py <delay (ms)> <executable> [args ...]")
+        exit()
+    delay_time = float(sys.argv[1]) * 0.001
+    time.sleep(delay_time)
+    subprocess.call(sys.argv[2:], stdout=sys.stdout, stderr=sys.stderr)
```

Comment:
ok, I was confused about the documentation on this page where it passes None to stdout and stderr by default. But the test still works when I remove these arguments.

https://docs.python.org/3.4/library/subprocess.html#subprocess.call